app/models/job_type.rb

```diff
@@ -3,4 +3,13 @@
 class JobType < ApplicationRecord
   belongs_to :user
   has_many :jobs
+
+  before_save :nullify_blanks
```

Comment:
Can you please move this above the associations to be consistent with the other models?